subsys/bluetooth/mesh/rpl.h

```diff
@@ -27,3 +27,5 @@ bool bt_mesh_rpl_check(struct bt_mesh_net_rx *rx,
 void bt_mesh_rpl_clear(void);
 void bt_mesh_rpl_update(struct bt_mesh_rpl *rpl,
 			struct bt_mesh_net_rx *rx);
+size_t bt_mesh_rpl_get(void (*cb)(const struct bt_mesh_rpl *rpl_entry));
```

Comment:
Won't `bt_mesh_rpl_iter` be more relevant name for this function?

Reply:
Yes, the current verb used doesn't seem right. I'd propose "foreach" since that's used elsewhere in the Mesh stack  (`bt_mesh_model_foreach()`, `bt_mesh_subnet_foreach()`, `bt_mesh_cdb_node_foreach()`, etc).